packages/ui/Avatars/index.tsx

```diff
@@ -12,21 +13,20 @@ export const PersonAvatar: React.FC<PersonAvatarProps> = ({ personId }) => {
 };
 
 interface ProfileAvatar {
-  userId: string;
   imageUrl?: string | null;
+  email: string;
 }
 
-export const ProfileAvatar: React.FC<ProfileAvatar> = ({ userId, imageUrl }) => {
-  if (imageUrl) {
-    return (
-      <Image
-        src={imageUrl}
-        width="40"
-        height="40"
-        className="h-10 w-10 rounded-full object-cover"
-        alt="Avatar placeholder"
-      />
-    );
-  }
-  return <Avatar size={40} name={userId} variant="bauhaus" colors={colors} />;
+export const ProfileAvatar: React.FC<ProfileAvatar> = ({ imageUrl, email }) => {
+  const imageSrc = imageUrl || `https://www.gravatar.com/avatar/${sha256(email)}?d=retro`;
```

Comment:
Gravatar allows for a fallback default image using the d URL parameter. We can also set Boring Avatar as the default by utilizing [their API service](https://boringavatars.com/#api-service).